LibreNMS/Alert/Transport/Elasticsearch.php

```diff
@@ -61,7 +61,7 @@ public function contactElasticsearch($obj, $opts)
         }
 
         if (! empty($opts['es_index'])) {
-            $index = strftime($opts['es_index']);
+            $index = date($opts['es_index']);
```

Comment:
This change might break elastic transport

Reply:
Maybe strtotime() is what was supposed to be here?

Reply:
No its to build a index name for elasticsearch

Reply:
The default string is the change above it